docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -54,27 +71,43 @@ VSOutput Main(VSInput input)
 }
 ```
 
-First off, we define 2 types, `VSInput` and `VSOutput` which represent the vertex shader's input and output.
+First off, we define 2 types, `VSInput` and `VSOutput` which represent the 
+vertex shader's input and output.
+
+The input is 2, `float3` (vector of 3 float components), the first is the 
+"position" field, which are coordinates ranging from [-1.0, 1.0], if the values 
+are outside this range they are clipped, and we won't see them on screen.
```

Comment:
This is entirely false, the input vertices have no requirement on position or any data at all really,  this clipping stage only happens on vertices from the SV_Position (output) after the vertex shader is ran, also when noting [-1.0,1.0] ranges please mention NDC / Normalized Device Coordinates and/or explain it